qiita_pet/templates/sitebase.html

```diff
@@ -145,8 +146,7 @@
                 <li><a href="/study/public/">View Public Studies</a></li>
               </ul>
             </li>
-            {% set user_tmp = handler.user() %}
-            {% set user_level = user_tmp.level if user_tmp is not None else None %}
+            {% set user_level = user.level %}
```

Comment:
We try to minimize database accesses from the template. Is there a way that you can have this information from the handler rather than accessing it from the template?